Book documentation (#12)
COURAGEOUS-isf/format: docs/src/basic_concepts/record_fields.md

## Record organization
The format uses various fields to organize and distinguish **records**
and **record sets**[^1]:
`name`, `uas_id` and `record_number`.

### Records
**Records** are used to store data both in **detection** as well as **tracking**.
Each record must have a unique `record_number` inside **detection** or **tracking**,

> meaning that the same `record_number` can appear in both without causing conflict.

Reviewer: ![image](https://github.com/COURAGEOUS-isf/format/assets/16354904/76768c40-7762-41f1-b4e9-39f75f1c0f2c)
Bold text is way too distracting, use it sparingly. Don't use it on detection/tracking here.

> Each record must have a unique `record_number` inside **detection** or **tracking**,
> meaning that the same `record_number` can appear in both without causing conflict.

Reviewer: ```suggestion
Each record must have a unique `record_number` per set, be it detection or tracking.
```

## Record organization

> The format uses different structures to organize and distinguish data.

Reviewer: This doesn't say much, might as well remove it

Author: I agree that it doesn't provide much, but won't it feel a bit blunt without some sort of brief text?

Reviewer: Then I suggest something like `most data is given as _records_, which combine with extra information to create _record sets_`.

### Records

> **Records** represent all the information related with an entity in a given time and point in space.

Reviewer: Point in space is optional in detection sets so say "in a given time and, when applicable, point in space" or something like that

> Each record must have a unique `record_number`.

Reviewer: This is already explained in the reference so this doesn't add much, might as well remove it too
